tests/search/components/TestSearchForm.js

```diff
@@ -7,24 +7,30 @@ const render = ReactTestUtils.renderIntoDocument;
 const findByTag = ReactTestUtils.findRenderedDOMComponentWithTag;
 
 describe('<SearchForm />', () => {
+  var onSearch;
+  var root;
+  var form;
+  var input;
+
+  beforeEach(() => {
+    onSearch = sinon.spy();
+    root = render(<SearchForm onSearch={onSearch} />);
+    form = findByTag(root, 'form');
```

Comment:
Could do -  I was going to remove the the additional refs. Do you think refs is better somehow?